[2025-04-09T08:35:29.451+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-04-09T08:35:29.491+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: init_elt_pipeline.upload_file_to_minio manual__2025-04-09T08:35:28.833258+00:00 [queued]>
[2025-04-09T08:35:29.498+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: init_elt_pipeline.upload_file_to_minio manual__2025-04-09T08:35:28.833258+00:00 [queued]>
[2025-04-09T08:35:29.499+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-04-09T08:35:29.513+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): upload_file_to_minio> on 2025-04-09 08:35:28.833258+00:00
[2025-04-09T08:35:29.519+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=322) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-09T08:35:29.520+0000] {standard_task_runner.py:63} INFO - Started process 334 to run task
[2025-04-09T08:35:29.519+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'init_elt_pipeline', 'upload_file_to_minio', 'manual__2025-04-09T08:35:28.833258+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/init_elt_pipeline.py', '--cfg-path', '/tmp/tmpkdlmtv36']
[2025-04-09T08:35:29.521+0000] {standard_task_runner.py:91} INFO - Job 14: Subtask upload_file_to_minio
[2025-04-09T08:35:29.538+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-04-09T08:35:29.571+0000] {task_command.py:426} INFO - Running <TaskInstance: init_elt_pipeline.upload_file_to_minio manual__2025-04-09T08:35:28.833258+00:00 [running]> on host 208e59669cb5
[2025-04-09T08:35:29.660+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='init_elt_pipeline' AIRFLOW_CTX_TASK_ID='upload_file_to_minio' AIRFLOW_CTX_EXECUTION_DATE='2025-04-09T08:35:28.833258+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-09T08:35:28.833258+00:00'
[2025-04-09T08:35:29.661+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-04-09T08:35:29.681+0000] {init_minio.py:24} INFO - Minio client connected successfully
[2025-04-09T08:35:29.696+0000] {extractor.py:75} INFO - File uploaded to Minio successfully
[2025-04-09T08:35:29.696+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-04-09T08:35:29.697+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-04-09T08:35:29.706+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=init_elt_pipeline, task_id=upload_file_to_minio, run_id=manual__2025-04-09T08:35:28.833258+00:00, execution_date=20250409T083528, start_date=20250409T083529, end_date=20250409T083529
[2025-04-09T08:35:29.734+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-04-09T08:35:29.753+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-09T08:35:29.758+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
